### PR TITLE
Update message-parser requirements files for cachito

### PR DESCRIPTION
**Pull request by donpenney**
Signed-off-by: Don Penney <dpenney@redhat.com>

### CODE REVIEW

File: message-parser/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,34 @@
-grpcio>=1.29.0
-protobuf>=3.17.3
+#
```
Comment (nishant-parekh) [inline message-parser/requirements.txt:1]: I think we need to keep the u/s requirements.txt file untouched and move all our files into a cachito directory.. that way both the u/s and d/s changes can persist in the repo

**Review by nishant-parekh — COMMENTED** [review]
just a comment to consolidate our changes in a separate directory so we donot override the u/s requirements.txt

File: message-parser/requirements.txt (hunk 1 above)
Comment (donpenney) [reply message-parser/requirements.txt]: Done

**Review by nishant-parekh — APPROVED** [review]

**Pull request merged**